### PR TITLE
KAFKA-17373: Add print.epoch to kafka-console-share-consumer.sh/kafka-console-consumer.sh

### PR DESCRIPTION
**Pull request by xijiu**
As title, setting `print.epoch=true` can show the `ConsumerRecord#leaderEpoch`

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java [hunk 1]
```diff
@@ -130,25 +134,31 @@ public void writeTo(ConsumerRecord<byte[], byte[]> consumerRecord, PrintStream o
                 } else {
                     output.print("NO_TIMESTAMP");
                 }
-                writeSeparator(output, printPartition || printOffset || printDelivery || printHeaders || printKey || printValue);
+                writeSeparator(output, anyTrue(printPartition, printOffset, printDelivery, printEpoch, printHeaders, printKey, printValue));
```
Comment (chia7712) [inline tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java]: Pardon me, what is the benefit of this helper?

**Review by chia7712 — COMMENTED** [review]
@xijiu thanks for your patch

File: tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java (hunk 1 above)
Comment (xijiu) [reply tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java]: When I coding style like this: `a | b | c | d | e ... `, then I received a checkstyle ERROR: 
```
[ant:checkstyle] [ERROR] /Users/likangning/xijiu/kafka/tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java:137:32: Boolean expression complexity is 6 (max allowed is 5). [BooleanExpressionComplexity]
```

I guess the boolean expressions are to much, but I'm not sure if this approach is reasonable

Comment (chia7712) [reply tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java]: How about adding `BooleanExpressionComplexity` to suppression?

Comment (xijiu) [reply tools/src/main/java/org/apache/kafka/tools/consumer/DefaultMessageFormatter.java]: > How about adding `BooleanExpressionComplexity` to suppression?

Wonderful idea, I have fixed it, PTAL   @chia7712

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request merged**